### DISCUSSION TITLE
Roborock T7S (S7) Miio rrcheck access rrcheck access denied

### DISCUSSION THREAD

**Discussion by ak4741a** [Hypfer/Valetudo#1765] [Q&A / Support] [answered]
Hi,

My Roborock T7S(S7) already installed  root & Valetudo
Valetudo home page show error loading map data
Check log 
`Miio error response {
  id: 1434,
  error: { code: -10002, message: 'rrcheck access denied' },
  exe_time: 1
}`

`StatisticsCapabilityRouter: Received error from robot while handling route "/" { body: {}, message: 'rrcheck access denied' }
`
But system information info is correct
Is there any step I'm missing?  thanks.

**Comment by Hypfer** [comment, marked as answer]
The T7S is not listed as a supported robot on valetudo.cloud

What you're seeing there is the region lock being unhappy with something about the robot.
